net/udp/udp.h

```diff
@@ -124,6 +124,17 @@ struct udp_conn_s
   int32_t  sndbufs;       /* Maximum amount of bytes queued in send */
   sem_t    sndsem;        /* Semaphore signals send completion */
 #endif
+#ifdef CONFIG_NETDEV_RSS
+  int      rcvcpu;        /* Last recvfrom cpuid */
```

Comment:
Please check code carefully to make sure it does not contain any redundant code.